apps/style/common.scss

```diff
@@ -530,6 +530,10 @@ html[dir='rtl'] div#visualizationResizeBar {
   cursor: default;
 }
 
+html[dir='rtl'] .karel-counter-text {
+  text-anchor: start;
+}
```

Comment:
nit; to match our other declarations of similar functionality, this should be nested inside the `.karel-counter-text` selector; 

```scss
 .karel-counter-text {
   text-anchor: end;
   font-size: 16px;
   font-family: Verdana, Geneva, sans-serif;
   font-weight: bold;
   fill: white;
   stroke: black;
   stroke-width: 1;
   cursor: default;

   html[dir='RTL'] & {
     text-anchor: start;
   }
}
```